src/docfx/build/moniker/MonikersProvider.cs

```diff
@@ -96,5 +96,17 @@ public List<string> GetZoneMonikers(string rangeString, List<string> fileLevelMo
             monikers.Sort(Comparer);
             return monikers;
         }
+
+        public string GetGroupIdFromMonikerList(IEnumerable<string> monikers)
```

Comment:
`IEnumerable` => `List`?

Reply:
What about `string HashUtility.GetMd5HashShortStable(List<string> items)`?

Reply:
that requires a comparer to be parsed into?

---
In reply to: [240448635](https://github.com/dotnet/docfx/pull/3750#discussion_r240448635) [](ancestors = 240448635)